src/layouts/Default.vue

```diff
@@ -1,18 +1,31 @@
 <template>
-  <div class="layout">
-    <header class="header">
-      <strong>
-        <g-link to="/">{{ $static.metadata.siteName }}</g-link>
-      </strong>
-      <nav class="nav">
-        <g-link class="nav__link" to="/">Home</g-link>
-        <g-link class="nav__link" to="/about/">About</g-link>
-        <g-link class="nav__link" to="/faq/">FAQ</g-link>
-        <g-link class="nav__link" to="/blog/">Blog</g-link>
-      </nav>
-    </header>
-    <slot/>
-  </div>
+    <div class="layout">
+        <header class="masthead">
+            <b-navbar class="container justify-content-center" toggleable="lg">
+                <b-navbar-brand href="#">
+                    <img src="assets/images/logos/colour_text.png" class="d-inline-block align-top" alt="GMN Logo">
+                  {{ $static.metadata.siteName }}
+                </b-navbar-brand>
+                <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>
+                <b-collapse id="nav-collapse" is-nav>
+                    <b-navbar-nav id="navbar-menu">
+                        <b-nav-item href="/about/">About</b-nav-item>
+                        <b-nav-item-dropdown text="Guides" right>
+                            <b-dropdown-item href="#">Application Guidelines</b-dropdown-item>
+                            <b-dropdown-item href="#">Mentroship Guidelines</b-dropdown-item>
+                        </b-nav-item-dropdown>
+                        <b-nav-item href="/faq/">FAQ</b-nav-item>
+                        <b-nav-item disabled>Project & Participants</b-nav-item>
```

Comment:
To which page should that link?  The mentee page?

Reply:
And where should be the mentors?

Reply:
It was supposed to be linked to `past_project.md` I previously pushed. But since we are grouping them by past Gmentees, we have to change the whole concept and I haven't done that. Another thing is since we will auto-generate the Gmentees, Gmentors, Network Manager, and project information, I will just add a few sentence to the `project & participants.md` such as Meet our participants. This page is coming soon, thanks. How is that?

Since you said having it in Openlifesci way is good, we will be opting for that. So the Gmentees, Gmentors, Network Manager, together with the project information will be on the same page.

Reply:
Sounds good,
You can already create the page, maybe `participants.md` (never spaces in page or file names) with the short text as you said.